otterdog/eclipse-cdt-cloud.jsonnet

```diff
@@ -5,6 +5,16 @@ local custom_branch_protection_rule(pattern) =
     required_approving_review_count: 0,
   };
 
+// for trace-related repos
+local custom_branch_protection_rule_trace(pattern) = 
+  orgs.newBranchProtectionRule(pattern) {
+    dismisses_stale_reviews: true,
+    is_admin_enforced: true,
+    required_approving_review_count: 1,
+    requires_status_checks: false,
+    requires_strict_status_checks: true,
```

Comment:
requiring strict status checks but not status checks does not really make sense, as the latter is only taken into account when the first one is enabled, you could also remove the strict status checks to avoid confusion.

Reply:
Thanks - will fix this

Reply:
done